assemblers/org.osbuild.qemu

```diff
@@ -338,7 +345,8 @@ def partition_from_json(js) -> Partition:
                   start=js.get("start"),
                   size=js.get("size"),
                   bootable=js.get("bootable"),
-                  name=js.get("name"))
+                  name=js.get("name"),
+                  uuid=js.get("uuid"))
```

Comment:
I think you could just use `**js` here.

Reply:
I initially wanted it to be explicit but it is indeed getting a bit long...